### ISSUE TITLE
Files in skfasttext directory

### ISSUE THREAD

**Issue by zebrolo** [tripl3a/hatespeech#1]
What is the purpose of the files in the [fkfasttext](https://github.com/tripl3a/hatespeech/tree/master/skfasttext) directory? What is the relation to the main script file?

**Comment by tripl3a** [comment]
Right now, the main script file HateSpeech.py works independently. But it produces the files train_data.txt and test_data.txt, which sk_ft_example.py uses, to train (fit) a model and predict labels/probabilities via an instance of SimpleFastTextClassifier.

SimpleFastTextClassifier is the class which implements the scikit-learn interfaces. It's a wrapper for using fastText within scikit. I will need this class for Bagging and you might need it for GridSearch. Probably there will be changes necessary for it to work with scikit, it's just a very first version.

I think SimpleFastTextClassifier should encapsulate all the communications with the fastText API. So we need to change the main script file and remove those parts there.

**Comment by zebrolo** [comment]
Ok, this means:
* `SimpleFastTextClassifier` - instantiates a FastText model
* `HateSpeech.py` - essentially provides a training set for fitting the model instantiated by `SimpleFastTextClassifier` and a validation set on which the label probabilities are computed
+ `sk_ft_example.py` - realizes the model fitting and label prediction, takes the files created by `SimpleFastTextClassifier`as input.

Have I missed anything?
